src/fsharp/FSharp.Build.netcore.nuget/layoutbuildnuget.fsx

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information,
+//
+// Layout the nuget package for the fsharp compiler
+//=========================================================================================
+open System.IO
+
+try
+    //=========================================================================================
+    // Command line arguments
+    //=========================================================================================
+
+    // Try head was introduced in F# 4.0
```

Comment:
tryHead is available in F# 4.0, and I assume this script is running as F# 4.0, so why do we need it here?